app/scripts/modules/kubernetes/src/pipeline/stages/runJob/configureJob.controller.js

```diff
@@ -133,6 +138,20 @@ module.exports = angular.module('spinnaker.kubernetes.pipeline.stage.runJobStage
       }
     };
 
+    this.setPostStartHandler = function(handler) {
```

Comment:
Use arrow functions here and below (especially since these functions use `this`).

Reply:
good catch. that was a copy/paste oversight!